### PR TITLE
feat: add "aks rotate-certs" command

### PR DESCRIPTION
**Pull request by andyzhangx**
add aks rotate-certs command

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@qwordy please take a look at this PR.

**Comment by andyzhangx** [comment]
/hold
hold on, `api-version=2019-10-01` is not ready on all regions now, need wait for a few days and rerun the failed tests

```
    return handle_exception(ex)
  File "/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/azure/cli/testsdk/patches.py", line 33, in _handle_main_exception
    raise ex
  File "/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 603, in execute
    raise ex
  File "/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 661, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
vcr.errors.CannotOverwriteExistingCassetteException: Can't overwrite existing cassette ('/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_blb_vmas.yaml') in your current record mode ('once').
No match for the request (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000001?api-version=2019-10-01>) was found.
Found 1 similar requests with 1 different matcher(s) :

1 - (<Request (PUT) https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000001?api-version=2019-08-01>).
Matchers succeeded : ['method', 'scheme', 'host', 'port', 'path']
Matchers failed :
_custom_request_query_matcher - assertion failure :
None

```

**Comment by sauryadas** [comment]
Thanks @andyzhangx .. this is a draft and will only be published when i sign off..

**Comment by andyzhangx** [comment]
@qwordy @yonzhan how can I rerun the failed tests?

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -56,22 +56,22 @@
...
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedCluster
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterAADProfile
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterAddonProfile
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterAgentPoolProfile
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterLoadBalancerProfile
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterLoadBalancerProfileManagedOutboundIPs
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterLoadBalancerProfileOutboundIPPrefixes
+from azure.mgmt.containerservice.v2019_10_01.models import ManagedClusterLoadBalancerProfileOutboundIPs
+from azure.mgmt.containerservice.v2019_10_01.models import AgentPool
+from azure.mgmt.containerservice.v2019_10_01.models import ResourceReference
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py]: Do you support multi-API? You use a static import from a certain API version.

**Comment by qwordy** [comment]
> 
> 
> @qwordy @yonzhan how can I rerun the failed tests?

You can use `azdev test <target> --lf` to rerun the failed tests in the previous run.
`azdev test <target>`, target can be module name, test case name. So you can run a single test case only.
You can also use IDE to run test cases.

**Comment by andyzhangx** [comment]
@qwordy this PR is ready to review and then merge since the related aks build has been rolled out to most regions.
cc @sauryadas

**Comment by qwordy** [comment]
> 
> 
> @qwordy this PR is ready to review and then merge since the related aks build has been rolled out to most regions.
> cc @sauryadas

OK.

**Comment by qwordy** [comment]
Does this command have any parameters?

**Comment by andyzhangx** [comment]
> Does this command have any parameters?

@qwordy No, it only has one confirmation msg, if user select Yes, then continue rotate certs.

**Comment by qwordy** [comment]
> 
> 
> > Does this command have any parameters?
> 
> @qwordy No, it only has one confirmation msg, if user select Yes, then continue rotate certs.

OK!

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 2]
```diff
@@ -829,6 +829,12 @@
     crafted: true
 """
 
+helps['aks rotate-certs'] = """
+    type: command
+    short-summary: Rotate certificates and keys on a managed Kubernetes cluster
+    long-summary: "Kubernetes will be unavailable during cluster certificate rotation."
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py]: Remove the double quotes?

Comment (andyzhangx) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py]: fixed

**Comment by qwordy** [comment]
Azure Pipeline upgrades python3 to use python3.8 today without asking us customers' ideas. So it fails. Python 3.8 has no time.clock(). We have fixed this compatibility problem in dev branch. Could you please merge dev branch so that the CI checks can use the latest code? Thanks.

**Comment by andyzhangx** [comment]
@qwordy thanks, so it's ready for merge?

**Comment by qwordy** [comment]
> 
> 
> @qwordy thanks, so it's ready for merge?

Merged.

**Pull request merged**